airflow/scripts/discord_scripts/discord_bot.py

```diff
@@ -0,0 +1,100 @@
+import asyncio
+from datetime import datetime, timedelta
+import discord
+from dotenv import load_dotenv
+import os
+
+intents = discord.Intents.default()
+intents.members = True
+
+load_dotenv()
+
+BOT_TOKEN = os.getenv('BOT_TOKEN')
```

Comment:
Where is your .env file ? are you hiding it with .gitignore? if so , where is the .gitignore update on the root?

Reply:
My .env is in the discord_scripts file, I added a .gitignore on the root